db/migrate/20150403010502_add_token_to_users.rb

```diff
@@ -0,0 +1,9 @@
+class AddTokenToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :token, :string
+  end
+
+  User.all.each do |user|
```

Comment:
I do not recommend this as in schema migration you are trying to do data migration. If you want to update data in database use custom rake tasks. Check them here http://guides.rubyonrails.org/command_line.html#custom-rake-tasks. It is easy to set it up.